resource/plugins/python/publisher/collectors/widget/nuke_movie_publisher_collector_options.py

```diff
@@ -396,21 +401,14 @@ def report_input(self):
             else:
                 message = 'No script node selected!'
         elif self._render_from_sequence_rb.isChecked():
-            num_objects = (
-                1
-                if self.image_sequence_path
-                and len(self.image_sequence_path) > 0
-                else 0
-            )
+            num_objects = 1 if self.image_sequence_path else 0
             if num_objects > 0:
                 message = '1 image sequence selected'
                 status = True
             else:
                 message = 'No image sequence selected!'
```

Comment:
```suggestion
            if self.image_sequence_path:
                message = '1 image sequence selected'
                status = True
            else:
                message = 'No image sequence selected!'
```
Same for the other num_objects. Doesn't really make sense.